packaging/installer/methods/offline.md

```diff
@@ -1,38 +1,85 @@
-# Install Netdata on offline systems
+# Install Netdata on Offline Systems
 
+This guide explains how to install Netdata Agent on systems without internet access.
 
-Our kickstart script enables offline Netdata Agent installation by prefetching required files (static installs only for now). Future support for local builds is possible, but no ETA. For offline systems, users can try their distro’s package tools (e.g., `apt-offline`), though it's not officially supported.
+Netdata supports offline installation of the Agent using our `kickstart.sh` script.
 
-## Preparing the offline installation source
+This method:
 
-The first step in installing Netdata on an offline system is preparing the offline installation source. This can be done as a regular user on any internet-connected system with the following tools available:
+- Downloads all required files on an internet-connected machine
+- Transfers these files to the offline system
+- Supports static builds only (currently)
+- Does not support automatic updates on offline systems
 
-- cURL or wget
-- sha256sum or shasum
-- A standard POSIX compliant shell
+---
 
-To prepare the offline installation source, simply run:
+## Step 1: Prepare the Offline Installation Package
+
+On your internet-connected machine, you'll need::
+
+| Requirement             | Purpose                       |
+|-------------------------|-------------------------------|
+| `curl` or `wget`        | Download the kickstart script |
+| `sha256sum` or `shasum` | Verify download integrity     |
+| POSIX-compliant shell   | Run the installation scripts  |
+
+Run the following command:
+
+- using `wget`
+  ```bash
+  wget -O /tmp/netdata-kickstart.sh https://get.netdata.cloud/kickstart.sh
+  sh /tmp/netdata-kickstart.sh --release-channel stable --prepare-offline-install-source ./netdata-offline
+  ```
+- or using `curl`
+  ```bash
+  curl https://get.netdata.cloud/kickstart.sh > /tmp/netdata-kickstart.sh
+  sh /tmp/netdata-kickstart.sh --release-channel stable --prepare-offline-install-source ./netdata-offline
+  ```
+
+  > [!NOTE]
```

Comment:
same as above, will stop commenting on all of these, just do a [! search and fix them :smile: